homeassistant/components/stream/worker.py

```diff
@@ -256,7 +257,7 @@ def reset(self, video_dts: int) -> None:
             input_astream=self._input_audio_stream,
         )
         if self._output_video_stream.name == "hevc":
-            self._output_video_stream.codec_tag = "hvc1"
+            self._output_video_stream.codec_context.codec_tag = "hvc1"
```

Comment:
The `__setattr__` of the `Stream` class passes unknown attributes directly to the `CodecContext` to be assigned there. I.e. both work.

In `13.1.0` only the latter has an explicit type annotation.